sdk/applicationinsights/arm-appinsights/test/sampleTest.spec.ts

```diff
@@ -30,16 +28,16 @@ const recorderOptions: RecorderStartOptions = {
 describe("My test", () => {
   let recorder: Recorder;
 
-  beforeEach(async function (ctx) {
+  beforeEach(async (ctx) => {
```

Comment:
Is changing from anonymous function to arrow function necessary?  We used to prefer functions for mocha due to https://mochajs.org/#arrow-functions.  Is vitest free from issue around `this`? If yes I'll update code gen for this and assertion below.

Reply:
Yes, the `this` is no longer needed and can be eliminated.  Should make allocations faster since there is no `this` binding necessary.